### DISCUSSION TITLE
step by step process to upgrade Snipe IT from Version v6.0.14 to V 6.2.0

### DISCUSSION THREAD

**Discussion by ravipawar23** [snipe/snipe-it#13712] [Q&A]
step by step process to upgrade Snipe IT from Version v6.0.14 to V 6.2.0

what is pross and cons 

what is the prerequisite to perform and mandatory precautions before proceeding further

**Comment by snipe** [comment]
We cover the different ways of upgrading here: https://snipe-it.readme.io/docs/upgrading

There are no new requirements introduced between v.6.0.14 and v6.2.1.

**Reply by ravipawar23** [reply]
i tried this method but my website crashed. can you please help me with more detailed method.

**Reply by marcusmoore** [reply]
@ravipawar23 Can you help us out by detailing the steps you took and what errors you received in the process?

**Reply by mauro-miatello** [reply]
I used the "php upgrade.php" method with no issue BUT I discovered that my application is in /var/www/snipe and the script needs 2 folders /var/www/.config and /var/www/.cache where it can write

**Reply by vithurshanselvarajah** [reply]
What is the permissions of the /var/www/snipe? 

I normally need to run as the apache user to upgrade so an example command would be 

sudo -u www-data php upgrade.php 

(replacing www-data with your web server user)

**Reply by mauro-miatello** [reply]
owner is www-data and I run the command as you with "sudo -u www-data" before

**Reply by vithurshanselvarajah** [reply]
Are there any logs in snipe/storage/logs?

Also in .env can you set debug to true (file will be in the root of snipe) and see if a error message shows when you try to access the site.
